.vib/suitecrm/cypress/cypress/integration/suitecrm_spec.js

```diff
@@ -1,48 +1,26 @@
 /// <reference types="cypress" />
 import { random } from '../support/utils';
 
-it('allows to log in/log out', () => {
-  cy.login();
-  cy.get('span[class="error"]').should('not.exist');
-  cy.contains('SUITECRM DASHBOARD');
-  cy.visit('/index.php?module=Users&action=Logout');
-  cy.contains('Log In');
-});
-
-it('allows to import accounts', () => {
+it('allows to import accounts and list them', () => {
   cy.login();
   cy.visit('/index.php?module=Accounts&action=index');
   cy.contains('Import Accounts').click({ force: true });
-  cy.get('input[type="file"]').selectFile('cypress/fixtures/accounts.csv');
-  cy.get('input#import_update').click();
-  cy.get('#gonext').click();
-  cy.contains('Here is how the first several rows');
-  cy.get('#gonext').click();
-  cy.contains('Check the mappings to make sure that they are what you expect');
-  cy.get('#gonext').click();
-  cy.contains('perform a duplicate check');
-  cy.get('#importnow').click();
-  cy.contains('View Import Result');
-  cy.contains('View Accounts').click({ force: true });
-  cy.get('[title="Edit"]').its('length').should('gt', 1);
-});
 
-it('allows adding a new user', () => {
-  cy.login();
-  cy.visit('index.php?module=Users&action=index');
-  cy.contains('Create New User').click({ force: true });
-  cy.fixture('users').then((user) => {
-    cy.get('#user_name').type(`${user.newUser.username}.${random}`);
-    cy.get('#first_name').type(`${user.newUser.firstName}.${random}`);
-    cy.get('#last_name').type(`${user.newUser.lastName}.${random}`);
-    cy.get('#Users0emailAddress0').type(`${user.newUser.email}.${random}`);
-    cy.get('#SAVE_FOOTER').click();
-    cy.get('#tab2').click();
-    cy.get('#new_password').type(`${user.newUser.password}.${random}`);
-    cy.get('#confirm_pwd').type(`${user.newUser.password}.${random}`);
-    cy.get('#SAVE_FOOTER').click({ force: true });
-    cy.contains('View Users').click({ force: true });
-    cy.contains(`${user.newUser.username}.${random}`);
+  const accountsFile = 'cypress/fixtures/accounts.csv';
+  cy.readFile(accountsFile).then((data) => {
+    const randomizedData = data.replace(/Test Account [a-z0-9_]+,/g, `Test Account ${random},`);
+    cy.writeFile(accountsFile, randomizedData);
+  });
+  cy.get('[type="file"]').selectFile(accountsFile);
+  cy.get('#import_update').click();
+  for (let i = 2; i < 5; i++) {
+    cy.get('#gonext').click();
+    cy.contains(`Step ${i}`);
+  }
+  cy.get('#importnow').click();
+  cy.contains('Exit').click({ force: true });
+  cy.get('.list').within(() => {
+    cy.contains(`${random}`);
```

Comment:
```suggestion
    cy.contains(`Test Account ${random}`);
```

Your check is correct, but I think it is neater to look for the whole thing we were using above. Feel free to skip the suggestion.